service/worker/service.go

```diff
@@ -394,18 +395,26 @@ func (s *Service) startIndexer() {
 	}
 }
 
-func (s *Service) startMigrationIndexer() {
-	visibilityIndexer := indexer.NewMigrationIndexer(
+func (s *Service) startMigrationDualIndexer() {
```

Comment:
nit: Do we need to do anything during shutdown? There's no stop exposed/called. This is a problem with rest of the components used in worker.

Reply:
This seems common problem for all the components in the workers, mostly only. call close when there is issue during start